examples/Linux/Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM debian:10 AS builder
+
+RUN apt-get update && apt-get install -y unzip libpcap-dev build-essential git cmake libpcap0.8 gdb
```

Comment:
Is gdb really needed?

Reply:
Ah! GDB was whilst I was debugging. I'll remove it.

Reply:
Glad I could help.